osu.Game/Screens/Play/HUD/LegacyComboCounter.cs

```diff
@@ -170,9 +170,9 @@ private void transformPopOut(int newValue)
             popOutCount.FadeTo(0.75f);
             popOutCount.MoveTo(Vector2.Zero);
 
-            popOutCount.ScaleTo(1, pop_out_duration, pop_out_easing);
-            popOutCount.FadeOut(pop_out_duration, pop_out_easing);
-            popOutCount.MoveTo(displayedCountSpriteText.Position, pop_out_duration, pop_out_easing);
+            popOutCount.ScaleTo(1, pop_out_duration);
+            popOutCount.FadeOut(pop_out_duration);
+            popOutCount.MoveTo(displayedCountSpriteText.Position, pop_out_duration);
```

Comment:
Introduces a new unused const.
Shall I revert this

Reply:
Removing the const here is fine. It can be added back if it's ever required.